blackbox/suites/fuse/modules/drive9_customer_kimi_perf.py

```
@@ -0,0 +1,884 @@
+                summary = self.latency_summary(stat_values)
+                rows.append({"section": "namespace", "scale": scale_id, "layout": layout, "op": "stat", "unit": "ms", **summary})
+                ctx.perf_values(f"{self.id}.namespace.{scale_id}.{layout}.stat", stat_values, "ms")
+        finally:
+            ctx.target.unmount(handle)
+        return rows
+
+    def measure_stat_samples(self, data_dir: Path, samples: int, raw_path: Path) -> list[float]:
+        files = list(data_dir.rglob("*.bin"))
+        if not files:
+            return []
+        rng = random.Random(9)
```

Ruff (0.15.17)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)